functions/consumer/mongodb-consumer/src/test/java/org/springframework/cloud/fn/consumer/mongo/MongoDbConsumerApplicationTests.java

```diff
@@ -42,6 +42,7 @@
  */
 @SpringBootTest(properties = {
 		"spring.data.mongodb.port=0",
+		"spring.mongodb.embedded.version=4.0.12",
```

Comment:
**[UNRELATED+BOOT26]** 
The default embedded MongoDB version has been [removed in 2.6](https://github.com/spring-projects/spring-boot/pull/27108).

There is probably a more general place to set this than here and in each test. 

There is probably an more general place to set this than here.